vue-components/src/styles/mixins/ButtonGroup.scss

```diff
@@ -0,0 +1,17 @@
+@mixin ButtonGroup( $class: '' ) {
+  & > :first-child #{$class} {
```

Comment:
I wonder, why are we targeting a class descendant here instead of the first element itself? Same for the other two.

Reply:
because the styles are on the `<label>` and not on the `<span>` that wraps the `<label>` and the `<input>`. And the styles must be on the `<label>` so they can depend on the state of the input.

Reply:
I think this might create an issue if later, as @SaiSan-WMDE hinted before, we might want to also combine these toggles with other non toggle buttons. In that case we will not be able to control the border radii of the toggle group. Although now that I think of it, combining regular buttons with the toggle buttons is going to be mighty confusing... @SaiSan-WMDE  did I misunderstand you?

Reply:
I don't actually think we discussed that! But yes, it is unlikely that toggle and regular buttons need to be grouped together. And in case a mixed group is needed in the future, we'd come up with a solution then.

Reply:
Also, note that the $class parameter is optional. If it is not provided, we would be just targeting the first element itself like in the example.